packages/manager/modules/pci/src/components/project/billing/resource-usage/index.html

```diff
@@ -19,8 +19,8 @@
     </oui-datagrid-column>
     <oui-datagrid-column
         data-title="::'pci_billing_private_registry_type' | translate"
-        data-property="name"
     >
+        {{ $ctrl.getType($row.name) }}
```

Comment:
```suggestion
        <span data-ng-bind=":: $ctrl..getType($row.name)"></span>
```

Reply:
is there a reason to encapsulate it in a span ?

Reply:
because it's the main way that we implement it.
So why -> because it's team taked way

And personally for me we have more control having it into span.